### PR TITLE
LY-121452 Asset Editor - support multiple asset editing with tabs

### PR DESCRIPTION
**Pull request by sphrose**

### CODE REVIEW

File: Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorTab.h [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+ * All or portions of this file Copyright (c) Amazon.com, Inc. or its affiliates or
+ * its licensors.
+ *
+ * For complete copyright and license terms please see the LICENSE at the root of this
+ * distribution (the "License"). All use of this software is governed by the License,
+ * or, if provided, by the license below or the license accompanying this file. Do not
+ * remove or modify any license notices. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ */
```
Comment (lawsonamzn) [inline Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorTab.h:11]: The license needs to be changed for this to be okay to commit to the repo.

Comment (lawsonamzn) [reply Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorTab.h:11]: I'm seeing the same problem on all these files.  Do we need to rebase on top of development before trying again?

Comment (AMZN-daimini) [reply Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorTab.h:11]: Would a rebase fix files that are being added in this PR?

**Comment by rainbj** [comment]
Can you please include a screenshot of the updated tabs?

File: Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorTab.h (hunk 1 above)
Comment (lawsonamzn) [reply Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorTab.h:11]: Yes, if you rebase it on top of upstream/development (assuming you have upstream configured).

Comment (sphrose) [reply Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorTab.h]: Fixed

**Comment by sphrose** [comment]
Moving PR to https://github.com/o3de/o3de/pull/2931. Leaving this open temporarily for reference. Will close when no longer needed.

**Pull request closed without merging**